cluster-autoscaler/cloudprovider/gce/gce_manager_test.go

```diff
@@ -1470,6 +1657,151 @@ func TestParseMIGAutoDiscoverySpecs(t *testing.T) {
 	}
 }
 
+func TestParseMIGAutoDiscoverySpec(t *testing.T) {
```

Comment:
Why is this (and the following 2 tests) not simply a part of `TestParseMIGAutoDiscoverySpecs`?